libcxx/cmake/config-ix.cmake

```diff
@@ -14,14 +14,6 @@ include(CheckCSourceCompiles)
 # link with --uwnindlib=none. Check if that option works.
 llvm_check_compiler_linker_flag(C "--unwindlib=none" CXX_SUPPORTS_UNWINDLIB_EQ_NONE_FLAG)
 
-if(WIN32 AND NOT MINGW)
```

Comment:
Why do you think we can remove this?

It seems to be disabling the linking of libc, I assume because we have to do something really weird in this case?

Is it possible that we're no accidentally trying to link libc? 

I guess if it passes CI, it's safe to ship.

Reply:
I removed this since `LIBCXX_HAS_C_LIB` had no uses anymore after my refactor above.

Reply:
If everything passes, I guess we're good? I